index.cjs

```diff
@@ -34,11 +34,11 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['*.ts'],
+      files: ["*.ts"],
```

Comment:
Interesting, the linter does not complain about formatting in the CI. We should have a look someday

Reply:
I think we don't include prettier in the lint checks

I will have a look